cluster/examples/kubernetes/ceph/object-openshift.yaml

```diff
@@ -42,6 +42,8 @@ spec:
       #target_size_ratio: ".5"
   # Whether to preserve metadata and data pools on object store deletion
   preservePoolsOnDelete: true
+  # createRoute creates route for ceph-rgw service
+  createRoute: true
```

Comment:
We need documentation for this in ceph-cluster-crd.yaml

Reply:
Also probably good to mention this is only applicable for OpenShift clusters